### PR TITLE
Quick fixes

### LINKED ISSUE #1373: TextRenderer.ShadowDrawer isn't entirely correct
It's used in `TextRenderer.drawLayer` to draw *all* text, not just shadowed text.

### LINKED ISSUE #1529: GuardianEntity#setTarget
GuardianEntity#setTarget(int progress) should be GuardianEntity#setTarget(int entityId)
https://discordapp.com/channels/507304429255393322/521545796882006027/727210582247473243

### LINKED ISSUE #1530: Tooltip methods in Block and Item have different names
Both methods append tooltip lines to a mutable `List<Text>`. The method in `Block` is called `buildTooltip`, while `Item` calls it `appendTooltip`. That leads to weird code such as this in `BlockItem`:

IMO the one in `Block` should be changed to `appendTooltip` to match items.

### PR DESCRIPTION
**Pull request by YanisBft**
Closes #1373 
Closes #1529 
Closes #1530 
Progress on #987

### CODE REVIEW

File: mappings/net/minecraft/client/font/TextRenderer.mapping [hunk 1]
```diff
@@ -178,7 +178,7 @@ CLASS net/minecraft/class_327 net/minecraft/client/font/TextRenderer
 		ARG 4 y
 		ARG 5 color
 		ARG 6 rightToLeft
-	CLASS class_5232 ShadowDrawer
+	CLASS class_5232 Drawer
```
Comment (Earthcomputer) [inline mappings/net/minecraft/client/font/TextRenderer.mapping:181]: This could be confused with the type of furniture. Maybe `Renderer`?

Comment (YanisBft) [reply mappings/net/minecraft/client/font/TextRenderer.mapping:181]: Could work too, will wait a bit for opinions before renaming it

Comment (YanisBft) [reply mappings/net/minecraft/client/font/TextRenderer.mapping:181]: Wouldn't it be weird to have a TextRenderer.Renderer class?

Comment (Juuxel) [reply mappings/net/minecraft/client/font/TextRenderer.mapping:181]: Maybe `TextDrawer` to not have confusion from a plain `Drawer`?

**Review by haykam821 — APPROVED** [review]

File: mappings/net/minecraft/client/font/TextRenderer.mapping (hunk 1 above)
Comment (YanisBft) [reply mappings/net/minecraft/client/font/TextRenderer.mapping:181]: Who will confuse an inner class of TextRenderer with a furniture?

Comment (Juuxel) [reply mappings/net/minecraft/client/font/TextRenderer.mapping:181]: Good question. :tiny_potato:

**Review by Gegy — APPROVED** [review]

**Review by Earthcomputer — APPROVED** [review]

**Pull request merged**